.github/workflows/replay-verify-mainnet.yaml

```diff
@@ -21,7 +21,7 @@ env:
 
 jobs:
   replay-transactions:
-    timeout-minutes: 720
+    timeout-minutes: 60
```

Comment:
can it finish in 60 minutes?

Reply:
yes for the mainnet but.. doesn't hurt to be the same timeout, it looks like. changed back.